superset/assets/src/components/RefreshLabel.jsx

```diff
@@ -33,6 +31,8 @@ class RefreshLabel extends React.PureComponent {
     this.state = {
       hovered: false,
     };
+    this.mouseOver = this.mouseOver.bind(this);
```

Comment:
Shouldn't need an event handler here. Can do the color change via css `:hover` like this: https://stackoverflow.com/questions/25770590/change-color-when-hover-a-font-awesome-icon

Reply:
problem is we need to apply `text-primary` and don't want to hard code the color. I guess I could use a `.less` file that would import the main less file